networkstatsquerier/fetcher/fetcher.go

```diff
@@ -39,7 +47,37 @@ func GetListOfEndpoints(filePath string) (ListOfEndpoints, error) {
 
 	byteValue, _ := ioutil.ReadAll(jsonFile)
 
-	err = json.Unmarshal([]byte(byteValue), &result)
+	err = json.Unmarshal([]byte(byteValue), &initialList)
+
+	// return nil if an empty endpoint list was passed in
+	if initialList == nil {
+		return result, nil
+	}
+
+	if err != nil {
+		return result, err
+	}
+
+	finalResult, err := formatList(initialList)
+
+	return finalResult, err
+}
+
+func formatList(initialList map[string]interface{}) (ListOfEndpoints, error) {
+	var endptList ListOfEndpoints
+	var errs error
+
+	// Cerner's top-level JSON field is "endpoints"
+	cernerList, ok := initialList["endpoints"]
+	if ok {
+		endptList, errs = CernerList{}.GetEndpoints(cernerList)
+	}
+
+	// Everything else should have a top level JSON field of "Entries"
+	defaultList, ok := initialList["Entries"]
```

Comment:
consider something like:

```
// cerner

// epic

// default
-> error handling if not formatted as expected for default
```

Should somewhere include documentation on what the `default` formatting should look like. Probably should be included in the README.